### DISCUSSION TITLE
Passing arguments to Timetable

### DISCUSSION THREAD

**Discussion by BKI92** [apache/airflow#22642] [General]
Good day.
My app automatically generate DAGs with using Timetable.
But I  ran into a problem, how to pass different parameters into  my custom Timetable classes, that implements WorkdayFixedTimesTimetable.

I try to pass params into init method of WorkdayFixedTimesTimetable(), when i create my new dag like this:
```
TEST_TIMES = ["12:00:00", "13:10:05", "17:15:10"]
with DAG(
    dag_id="test_workday_fixed_times",
    start_date=datetime.datetime(2022, 3, 23),
    timetable=WorkdayFixedTimesTimetable(TEST_TIMES),
    tags=["test", "timetable"],
    catchup=False,
) as dag:
```

In WorkdayFixedTimesTimetable(Timetable) init method like:
```
class WorkdayFixedTimesTimetable(Timetable):
    def __init__(self, start_times: List[str]) -> None:
        self.start_times = start_times
        super(WorkdayFixedTimesTimetable, self).__init__()
```
        
But when i try to trigger my dag with airflow UI, I have a error 
"TypeError: __init__() missing 1 required positional argument: 'start_times'"
        
P.S.:  Ideally, I would like to implement a class WorkdayFixedTimesTimetable into which I can pass various parameters, and it, in turn, generates its own schedule based on the parameters entered.

**Comment by BKI92** [comment]
I don't know, how i didn't see it before creation issue, but it case  is covered in the documentation on this link 
https://airflow.apache.org/docs/apache-airflow/2.2.0/howto/timetable.html#parameterized-timetables

**Comment by arunporwal** [comment]
@BKI92: Did your plugin work? I am trying the same but it's not working for me. any example? I followed the document example.  Any other reference?
